### ISSUE TITLE
[Feature]: Ability to remove incognito mode in group once enabled

### ISSUE THREAD

**Issue by Mikaela** [simplex-chat/simplex-chat#2690]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Platform

Android, iOS

### App version

5.2

### Feature

I have joined some groups incognito, but I have found myself wishing for the option of becoming myself rather than staying incognito.

This comes up mainly in two situations:

- I am offered administrative access in group and I would like people to know who I am if I am going to have power over them.
- My actions outside of the group become relevant such as I want to refer to my blog posts or issues I have opened at GitHub or otherwise refer to the real me.
  - The situation in this case is that I am a RandomName and see the incognito symbol on top, but everyone following the group knows it's me.

Rejoining the group without an incognito mode would be an option if it didn't result in an error of being already joined to the group or being already connected to X assuming that the group link was shared publicly e.g. in the welcome message.

This is practically the opposite of https://github.com/simplex-chat/simplex-chat/issues/1896

**Comment by epoberezkin** [comment]
Yes, that's certainly a consideration - we discussed it in the context of revealing the main profile to incognito contacts.

The only question is whether it's worth doing now or as part of groups/communities redesign.

If we do it, it would be an irreversible action, having different profiles to different members is a bad idea...

Off topic - I'd like to ask your help in designing the next generation groups/communities product - so we have an MVP that works.

**Comment by epoberezkin** [comment]
Let me know if you can/want to participate in it, so we make a product that works for you and other community founders/owners/administrators - I think you can be a good proxy to what people like you need.

**Comment by Mikaela** [comment]
> The only question is whether it's worth doing now or as part of groups/communities redesign.

I don't think it's so important right now as groups have more serious issues such as not seeing all the other members or lack of moderation tooling (or the question of how to know the incognito who just joined isn't the other incognito that was just removed from group).

> Let me know if you can/want to participate in it, so we make a product that works for you and other community founders/owners/administrators - I think you can be a good proxy to what people like you need.

We discussed this a bit on SimpleX itself and I am definitely interested while I am not sure how qualified I am to represent such as diverse group of people. _(I am just leaving this comment here so it doesn't look like I am ignoring you while this paragraph is nothing new itself.)_

**Comment by epoberezkin** [comment]
> the question of how to know the incognito who just joined isn't the other incognito that was just removed from group

no way we can or want to answer that question, that's the point.

The solution is to start all new members with limited privileges and zero reputation scores, and once we have optional identities members with identities can be started with somewhat higher privileges. We're talking 2024 here :)
